### PR TITLE
Fix a number of spelling issues

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: pkgs/test/test/runner/json_reporter_utils.dart [hunk 1]
```diff
@@ -5,10 +5,9 @@
 import 'dart:convert';
 
 import 'package:path/path.dart' as p;
-import 'package:test_descriptor/test_descriptor.dart' as d;
```
Comment (jakemac53) [inline pkgs/test/test/runner/json_reporter_utils.dart:8]: Are we no longer putting imports from other packages in a different section?

Comment (kevmoo) [reply pkgs/test/test/runner/json_reporter_utils.dart:8]: Just using tool sorting seems like the most stable, canonical thing to do!

Comment (jakemac53) [reply pkgs/test/test/runner/json_reporter_utils.dart:8]: I don't see an opinion here https://dart.dev/guides/language/effective-dart/style#ordering so I am fine with this, just curious more than anything? I thought we used to recommend it separating them, but it always felt a bit arbitrary imo.

**Review by jakemac53 — APPROVED** [review]

File: pkgs/test/test/runner/json_reporter_utils.dart (hunk 1 above)
Comment (jakemac53) [reply pkgs/test/test/runner/json_reporter_utils.dart:8]: Ahah! https://github.com/dart-lang/site-www/pull/2405 :)

**Pull request merged**